src/js/footer/helper/index.js

```diff
@@ -192,38 +192,38 @@ export const loginUrl = () => {
   return `${window.getOauthUrl()}/oauth2/authorize?app_id=${getDomainAppID()}&l=${language}&brand=deriv${affiliate_token_link}${cookies_link}&platform=${sub_url}`;
 };
 
-const handleOutSystemsRedirection = () => {
-  const currentDomain = window.location.hostname;
-  let env;
-  if (
-    currentDomain === "deriv.com" ||
-    currentDomain === "deriv.be" ||
-    currentDomain === "deriv.me"
-  ) {
-    env = "production";
-  } else if (currentDomain === "staging.deriv.com") {
-    env = "staging";
-  } else {
-    env = "development";
-  }
-  switch (env) {
-    case "production":
-      return "https://hub.deriv.com/tradershub/preload-helper";
-    case "staging":
-      return "https://staging-hub.deriv.com/tradershub/preload-helper";
-    default:
-      return "https://hub.deriv.com/tradershub/preload-helper";
-  }
-};
-
-const isUserLoggedIn = !!getCookieByKey(document.cookie, "client_information");
-if (!isUserLoggedIn) {
-  var iframe = document.createElement("iframe");
-  iframe.src = handleOutSystemsRedirection();
-  iframe.style.width = "0";
-  iframe.style.height = "0";
-  iframe.style.border = "none";
-  iframe.style.visibility = "hidden";
-  iframe.setAttribute("sandbox", "allow-same-origin allow-scripts");
-  document.body.appendChild(iframe);
-}
+// const handleOutSystemsRedirection = () => {
```

Comment:
Go ahead and remove the code, no need to keep commented code. we can always get them back through GH history.